turbo/engineapi/engine_block_downloader/core.go

```diff
@@ -102,11 +102,6 @@ func (e *EngineBlockDownloader) download(ctx context.Context, hashToDownload lib
 		return
 	}
 	tx.Rollback() // Discard the original db tx
-	if err := e.insertHeadersAndBodies(ctx, tmpTx, startBlock, startHash, endBlock); err != nil {
```

Comment:
Call is removed, since insertion of blocks was performed in the call to `downloadAndLoadBodiesSyncronously`